bears/js/JSStandardBear.py

```diff
@@ -0,0 +1,28 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='standard',
+        output_format='regex',
+        output_regex=r'\s*(?P<filename>[^:]+):(?P<line>\d+):(?P<column>\d+):'
```

Comment:
Please dont capture `filename`. It isnt used.
Aldo if you copied this from a guide, it needs to be fixed.